docs/data-sources/clinvar.mdx

```diff
@@ -106,6 +106,14 @@ We only use the field with Type="Preferred". Multiple phenotypes may be reported
 - If a required allele is not available, we extract it from the reference sequence.
 - Only variants having a dbSNP id are extracted.
 - Note that a ClinVar accession may have multiple variants associated with it (possible in different locations)
+- VariantType is extracted from the Measure attributes.
+- Note that we currently don't support the following variant types:
+  - Microsatellite
+  - protein only
+  - fusion
+  - Complex
+  - Variation
+Translocation
```

Comment:
```suggestion
- VariantType is extracted from the Measure attributes.
- Note that we currently don't support the following variant types:
  - Microsatellite
  - protein only
  - fusion
  - Complex
  - Variation
  - Translocation
```

Reply:
@rajatshuvro Should this be here or under known issues?

Reply:
nice catch